README.md

```diff
@@ -49,6 +49,20 @@ Share Social , Sending Simple Data to Other Apps
 6. Make your `Application` class implements `ShareApplication`
  - Make `getFileProviderAuthority` function return the `android:authorities` that was added on AndroidManifest file
 
+Example: 
+```
+class MyApplication extends Application implements ShareApplication, ReactApplication {
+
+...
+
+     @override
+     String getFileProviderAuthority() {
+            return <File provider authority>
```

Comment:
could you provide an example of this?

Reply:
this string should be the same you added in the AndroidManifest. If a add a fixed string on the example can be misleading

Reply:
got it